2015-10-27.md

```diff
@@ -0,0 +1,32 @@
+Hi Tesselators!
+
+It's been a few weeks since the last This Week in Tessel, our apologies! We've been hard at work on the Tessel repos, building and growing the project.
```

Comment:
What do you mean re website update? The blog?
It's unfortunately pretty manual; Typically, I write this, test send, send, copy over to Tumblr, heroku restart tessel.io, and then it's live. Then Twitter etc.
Someday we'll fix that process.

In the meanwhile, updates are always welcome as issues or PRs on this repo, then I can funnel them through the right channels.